libs/designsystem/src/lib/components/chart-wip/chart-wip.types.ts

```diff
@@ -0,0 +1,13 @@
+import { ChartDataset as ChartJSDataset } from 'chart.js';
+
+export enum ChartType {
+  column = 'column',
+  bar = 'bar',
+}
+
+export type ChartDataset = ChartJSDataset<'bar'>;
+export type ChartData = ChartDataset[] | number[];
+
+export function isNumberArray(value: any): value is number[] {
```

Comment:
Is there a better place to store type helpers like this one?

Reply:
As long as it is only used by the chart, I think this is an ok place. We can move it to /helpers/ if we ever reuse it in another context.